### PR TITLE
Now supporting PySide2 when PySide is not present in the SG Desktop P…

### PR DESCRIPTION
**Pull request by lassond**
`import PySide.QtGui` was raising an exception on Fabio's machine

### CODE REVIEW

File: env/includes/settings/tk-multi-launchapp.yml [hunk 1]
```diff
@@ -65,7 +65,7 @@ settings.tk-multi-launchapp.motionbuilder:
 # setup_ue_project
 settings.tk-multi-launchapp.setup_ue_project:
   menu_name: Setup UE Project
-  icon: "{config_path}/icons/tools.png"
+  icon: "{config_path}/icons/imgspc.png"
```
Comment (lassond) [inline env/includes/settings/tk-multi-launchapp.yml:68]: Simon wants the company logo as the logo

File: hooks/tk-multi-launchapp/python_launch.py [hunk 2]
```diff
@@ -70,6 +70,11 @@ def execute(
         # where spaces can get in the full path name. Surround the argument with
         # quotes
         app_args = '"{}"'.format(app_args)
+        self.logger.info(
+            'Launching command: "{}\python.exe" {}'.format(
+                python_interpreter_dir, app_args
+            )
+        )
```
Comment (lassond) [inline hooks/tk-multi-launchapp/python_launch.py:77]: This makes it much easier to debug when exceptions are silent like for here

File: apps/setup_ue_project.py [hunk 3]
```diff
@@ -4,7 +4,19 @@
...
+    from PySide2.QtWidgets import QApplication, QFileDialog, QMessageBox
+
+    # Mandatory Application
+    app = QApplication()
+except:
+    from PySide.QtGui import QApplication, QFileDialog, QMessageBox
+
+    # Mandatory Application
+    app = QApplication("")
+
```
Comment (baptisteguidoux) [inline apps/setup_ue_project.py:19]: If you have acces to an `sgtk` instance at this moment, you could use `sgtk. has_qt5` or  `sgtk.has_qt4`. Qt should then be available in `sgtk.platform.qt` or `sgtk.platform.qt5`.

**Review by baptisteguidoux — APPROVED** [review]
Approved

File: apps/setup_ue_project.py (hunk 3 above)
Comment (lassond) [reply apps/setup_ue_project.py:19]: Good pointer, thanks!
I don't have an sgtk instance, I cannot even `import sgtk`

I will leave it like that

**Pull request merged**